classes/class-wpcom-liveblog-schema.php

```diff
@@ -108,11 +107,18 @@ public function get_live_blog_updates() {
 	 * @return string
 	 */
 	public function get_start_time( $entries_asc ) {
-		if ( ! isset( $entries_asc[0] ) || ! isset( $entries_asc[0]['datePublished'] ) ) {
+
+		if ( ! is_array( $entries_asc ) || empty( $entries_asc ) ) {
+			return '';
+		}
+
+		$first_value = array_shift( $entries_asc );
+
+		if ( ! empty( $first_value['datePublished'] ) ) {
```

Comment:
I guess that this should had been `if ( empty( $first_value['datePublished'] ) ) {` (`!` is extra)

Reply:
😨

Reply:
ehm, unit tests, ehm 😉